connector_voicent_helpdesk_ticket/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2019 Open Source Integrators
+# <https://www.opensourceintegrators.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import io
+import base64
+from odoo import api, models, _
+from ...queue_job.job import job
+from odoo.tools import pycompat
+from ...connector_voicent.examples import voicent
+import tempfile
+import shutil
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = 'helpdesk.ticket'
+
+    @job
+    @api.multi
+    def check_status_job(self, voicent_campaign, helpdesk_ticket,
+                         call_line):
+        for rec in self:
+            voicent_obj = voicent.Voicent(
+                call_line.backend_id.host,
+                str(call_line.backend_id.port),
+                call_line.backend_id.callerid,
+                str(call_line.backend_id.line))
+            status = voicent_obj.checkStatus(voicent_campaign)
```

Comment:
@guewen How can we get this job to be retried if the status returned by Voicent is not OK?